include/ada/url.h

```diff
@@ -268,9 +277,10 @@ namespace ada {
      * We assume that the input does not contain spaces or tabs
      * within the ASCII digits.
      * It returns how many bytes were consumed when a number is successfully parsed.
+     * On failure, it returns zero.
      * @see https://url.spec.whatwg.org/#host-parsing
      */
-    ada_really_inline size_t parse_port(std::string_view view) noexcept {
+    ada_really_inline size_t parse_port(std::string_view view, bool check_trailing_content) noexcept {
```

Comment:
Can we have a default value of `false` for `check_trailing_content`

Reply:
Done.